src/pages/legal-disclamer.html

```diff
@@ -71,4 +71,4 @@ <h2 class="section-title">Legal disclaimer</h2>
   <script src="js/legal-disclaimer.js"></script>
 </body>
 
```

Comment:
The same as before, this page should now be a route of the Gatsby application and this static file should be deleted.

Reply:
Added as a page.